hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -828,7 +843,7 @@ async def cancel_all(self, timeout_seconds: float):
                 cancel_payloads.append({
                     "id": ascend_ex_utils.uuid32(),
                     "orderId": order.exchange_order_id,
-                    "symbol": await OBDS.exchange_symbol_associated_to_pair(order.trading_pair),
+                    "symbol": await AscendExAPIOrderBookDataSource.exchange_symbol_associated_to_pair(order.trading_pair),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols